packages/truffle-db/src/workspace/pouch.ts

```diff
@@ -50,14 +53,34 @@ export class Workspace {
     return savePath;
   }
 
+  jsondownpouch(opts:any, callback:any):any {
+    // console.debug("options?? %o", opts);
+    const _opts = pouchdbUtils.assign({
+      db: jsondown.default
+    }, opts);
+
+    CoreLevelPouch.call(this, _opts, callback);
+  }
+
+  adapter(PouchDB:any):any {
+    PouchDB.adapter('jsondown', this.jsondownpouch, true);
+  }
+
   private ready: Promise<void>;
 
-  constructor () {
-    PouchDB.plugin(PouchDBMemoryAdapter);
+  constructor (workingDirectory:string) {
     PouchDB.plugin(PouchDBFind);
 
+    this.jsondownpouch["valid"] = () => true;
+    this.jsondownpouch["use_prefix"] = false;
```

Comment:
I think these should be in `this.adapter`

Reply:
I tried moving it these into this.adapter as we discussed and for some reason a bunch of errors got thrown about reading the json data. Not sure why - ok to leave it as is or would you rather I investigate further?